src/src/com/microsoft/aad/adal/DefaultTokenCacheStore.java

```diff
@@ -287,14 +287,15 @@ public void removeAll() {
      */
     @Override
     public ArrayList<TokenCacheItem> getTokensForUser(String userid) {
```

Comment:
overall: All interfaces should use generic interface like `List<>` or `Set<>` and not concrete implementations